samples/ImplicitFlow/AuthorizationServer/Startup.cs

```diff
@@ -0,0 +1,156 @@
+using System.Linq;
+using Microsoft.AspNetCore.Builder;
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.Extensions.Logging;
+using Microsoft.AspNetCore.Identity.EntityFrameworkCore;
+using Microsoft.Extensions.DependencyInjection;
+using CryptoHelper;
+using OpenIddict;
+using Microsoft.Extensions.Configuration;
+using System;
+using Mvc.Server.Models;
+using Mvc.Server.Services;
+
+namespace Mvc.Server
+{
+    public class Startup
+    {
+        public void ConfigureServices(IServiceCollection services)
+        {
+            var configuration = new ConfigurationBuilder()
+                        .AddJsonFile("config.json")
+                        .AddEnvironmentVariables()
+                        .Build();
+
+            services.AddDbContext<ApplicationDbContext>(options =>
+            {
+                options.UseSqlite("Filename=./authorization.db");
+            });
+
+            services.AddIdentity<ApplicationUser, IdentityRole<Guid>>()
+                .AddEntityFrameworkStores<ApplicationDbContext, Guid>()
+                .AddDefaultTokenProviders();
+
+            // Register the OpenIddict services, including the default Entity Framework stores.
+            services.AddOpenIddict<ApplicationUser, IdentityRole<Guid>, ApplicationDbContext, Guid>()
+
+                // Enable the authorization, logout, userinfo, and introspection endpoints.
+                .EnableAuthorizationEndpoint("/connect/authorize")
+                .EnableLogoutEndpoint("/connect/logout")
+                .EnableUserinfoEndpoint("/connect/userinfo")
+                .EnableIntrospectionEndpoint("/connect/introspect")
+
+                // Note: the Mvc.Client sample only uses the authorization code flow but you can enable
+                // the other flows if you need to support implicit, password or client credentials.
+                .AllowImplicitFlow()
+
+                // During development, you can disable the HTTPS requirement.
+                .DisableHttpsRequirement()
+
+                // Register a new ephemeral key, that is discarded when the application
+                // shuts down. Tokens signed using this key are automatically invalidated.
+                // This method should only be used during development.
+                .AddEphemeralSigningKey();
+
+            services.AddCors();
+            services.AddMvc();
+
+            services.AddTransient<IEmailSender, AuthMessageSender>();
+            services.AddTransient<ISmsSender, AuthMessageSender>();
+        }
+
+        public void Configure(IApplicationBuilder app, IHostingEnvironment env, ILogger<Startup> logger)
+        {
+            // Set the Environment e.g. via PowerShell> $env:ASPNETCORE_ENVIRONMENT="Development"
+            logger.LogInformation(env.EnvironmentName);
```

Comment:
This info is automatically logged for you by the hosting stack. Remove that please.